src/App.jsx

```diff
@@ -14,33 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SortType = {
+  ALPHABET: 'alphabet',
+  LENGTH: 'length',
+  DEFAULT: 'default',
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const handleSort = (type, setGoods, isReversed) => {
```

Comment:
```suggestion
const handleSort = (goods, { isReversed, sortType }) => {
```
- The utility function shouldn't know anything about the state, just gets goods, sorts and returns them
- utils should be a pure functions https://www.geeksforgeeks.org/pure-functions-in-javascript/